doc/ref_model/chapters/chapter07.rst

```diff
@@ -394,52 +394,116 @@ ZTA principles applied to Cloud infrastructure components are the following:
+`Defending against SolarWinds attacks 
+<https://www.techtarget.com/searchsecurity/news/252494495/Defending-against-SolarWinds-attacks-What-can-be-done>`_.
+Open-source code weaknesses can also be utilised by attackers, the Log4J vulnerability, impacting lots of 
+applications, is a recent example in this field. The cyber security of software supply chain are often not 
+taken into account. Governments are alerting and requesting actions to face these risks. The British 
+government is hardening the law and standards of cyber security for the supply chain. The US government 
+requested actions to enhance the software supply chain security. The security of software supply chain 
+is a challenge pointed out by the European Network and Information Security Agency, ENISA, 
+in the report `NFV Security in 5G - Challenges and Best Practices 
+<https://www.enisa.europa.eu/publications/nfv-security-in-5g-challenges-and-best-practices>`_.
+ 
+
+Software security
+~~~~~~~~~~~~~~~~~
 
-Software supply chain safety is crucial and can be a complex task in virtualised and containerised environments. Open source code is present in Cloud Infrastructure software from host Operating System to virtualisation layer components, the most obvious being represented by Linux, KVM, QEMU, OpenStack, and Kubernetes. Workloads components can also be composed of open source code. The proportion of open source code to an application source code can vary. It can be partial or total, visible or not. Open source code can be upstream code coming directly from open source public repositories or code within a commercial application or network function. To ensure the security of the whole system, all software and hardware components must reach the same level of security by following best security practices including secure lifecycle management. The SAFECode paper “Managing Security Risks Inherent in the Use of Third-party Components” provides a detailed risk management approach.
+Software supply chain safety is crucial and can be a complex task in virtualised and containerised environments
+with a greater attack surface. All software components must be trusted, from commercial software, open-source 
+code to proprietary software. The SAFECode paper “Managing Security Risks Inherent in the Use of Third-party 
+Components” provides a detailed risk management approach.
 
 To secure software code, the following methods must be applied:
 
--  Use best practices coding such as design pattern recommended in the `Twelve-Factor App <https://12factor.net/>`__ or `OWASP “Secure Coding Practices - Quick Reference Guide” <https://owasp.org/>`__
--  Require suppliers to provide a Software Bill of Materials to identify the open source modules in their product’s software releases
+-  Use best practices coding such as design pattern recommended in the `Twelve-Factor App <https://12factor.net/>`_ or `OWASP “Secure Coding Practices - Quick Reference Guide” <https://owasp.org/>`_
```

Comment:
how do these best practices help with supply chain security?

Also, can the bullets be re-ordered?

Reply:
List re-ordered